db/migrations/20181023141856_create_vow_flog.sql

```diff
@@ -14,6 +15,8 @@ CREATE INDEX vow_flog_log_index
     ON maker.vow_flog (log_id);
 CREATE INDEX vow_flog_header_index
     ON maker.vow_flog (header_id);
+CREATE INDEX vow_flog_msg_sender_index
+    ON maker.vow_flog (msg_sender);
```

Comment:
Super small thing - but I wonder if we want to remove the `DROP INDEX` statements below, or add `DROP INDEX maker.vow_flog_msg_sender_index` for consistency.

I believe that all indexes are dropped when the associated table is dropped, so it's probably safe to just remove those DROPs.

Reply:
👍 done.